app/views/layouts/_header.html.erb

```diff
@@ -1,8 +1,10 @@
 <nav class="">
     <ul>
         <li> <%= link_to "Store Home", store_path %> </li>
-        <% if !!@cart %>
-          <li> <%= link_to "Cart", cart_path(current_cart) %></li>
+        <% #binding.pry %>
+        <% if !!current_user && !!current_user.current_cart %>
```

Comment:
I'm a little confused by the double-negation here. I think we can just do `current_user && current_user.current_cart`, right?

(I could be missing something or letting my style preferences show through too much: my reasoning is that we want the `current_user.current_cart` object if it exists, so we don't need to coerce to booleans.)